noir/scripts/test_native.sh

```diff
@@ -14,4 +14,6 @@ RUSTFLAGS=-Dwarnings cargo clippy --workspace --locked --release
 ./.github/scripts/cargo-binstall-install.sh
 cargo-binstall cargo-nextest --version 0.9.67 -y --secure
 
+# See https://github.com/AztecProtocol/aztec-packages/pull/10080
+RUST_MIN_STACK=8388608
```

Comment:
I expected this would have to be either:
* `export RUST_MIN_STACK=8388608` to affect all subsequent commands 
* `RUST_MIN_STACK=8388608 \` to treat it as a two-line command 
* `RUST_MIN_STACK=8388608 cargo nextest ...` to affect the cargo command only

Surprised it works 🤔

Reply:
Ah, I ignored the test in an earlier commit which is why it's working. Let's merge this anyway.

Reply:
Just wanted to make sure it doesn't work by accident. The [examples](https://www.baeldung.com/linux/set-env-variables-bash-command) are all suggesting it's just a variable in this case. Oh well.